PyYapf.py

```diff
@@ -190,6 +178,40 @@ def __exit__(self, type, value, traceback):
         if self.custom_style_fname:
             os.unlink(self.custom_style_fname)
 
+    def find_yapf(self):
+        """Find the yapf executable."""
+        # default to what is in the settings file
+        cmd = self.get_setting("yapf_command")
+        if not cmd:
```

Comment:
It's perhaps strange that we still show an error message in this case. I would actually suggest to remove the default `yapf_command' and make this case (when `cmd` is empty) the only case that the "search common possibilities" logic is invoked.

Reply:
It is a weird edge case; hitting it means the user edited the settings file and removed the default yapf_command (or otherwise made it evaluate as false).

reg. triggering on '' (and making that the default) -- perfect.  That takes care of the new user issue (which is the real issue) without adding magic for the "normal" case.  Maybe tweak the fail string to hint to try "" as yapf_command to trigger a search.